analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -207,33 +196,60 @@ class Bundler(
         }
     }
 
-    private fun parseProject(workingDir: File) =
+    private fun resolveGemsMetadata(workingDir: File): MutableMap<String, GemSpec> {
+        requireLockfile(workingDir) { workingDir.resolve("Gemfile.lock").isFile }
```

Comment:
Technically the lock file is not required, since Bundler can resolve the dependencies without it. Naturally, analyzer runs might differ if run at a later time. Maybe you have an opinion about that